modules/lib/lib-auth/src/main/resources/lib/xp/auth.ts

```diff
@@ -163,7 +163,7 @@ interface GetUserHandler {
  *
  * @returns {User} Information for logged-in user.
  */
-export function getUser(params: GetUserParams): User {
+export function getUser(params ?:GetUserParams): User {
```

Comment:
Same problem with annotation as the one mentioned in the other PR. Please don't change it.

Reply:
That's up to ye

Reply:
Yes, but it's you who are changing the annotation. So, change it back please and make the parameter optional.